packages/costa/src/runtime.ts

```diff
@@ -128,6 +135,22 @@ async function fetchPackageJsonFromTarballStream(fileStream: NodeJS.ReadableStre
   return JSON.parse(packageJson);
 }
 
+async function fetchMd5(filePath: string): Promise<string> {
+  return new Promise((resolve, reject) => {
```

Comment:
Is there necessary to use the stream API? It seems that the file might be a small package.json file thus digest() looks simple?

Reply:
the file is the whole npm package and might be quite large?